assignments/callbacks.js

```diff
@@ -2,29 +2,69 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+return cb(arr);
```

Comment:
This should be written as `return cb(arr[0]);` without the need for `function getFirstItem(items) {...}`. This applies throughout the rest of your code. I believe we talked about this and am not sure if you have updated it since then.